module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,33 @@
+#include "vowel.hpp"
+
+std::vector<char> vowels = {'a', 'e', 'i', 'o', 'u', 'y', 'A', 'E', 'I', 'O', 'U', 'Y'};
+
+bool isVowel(char &character)
+{
+    if (std::find(vowels.begin(), vowels.end(), character) != vowels.end())
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+
+void removeVowels(std::vector<std::string> &texts)
+{
+    for (auto &text : texts)
+    {
+        std::string textWithoutVowels = "";
+
+        for (auto &character : text)
```

Comment:
reference to char is 4 or 8 byte size, so in my opinion you should delete this reference and pass char by copy because normal char is 1 byte size
```suggestion
        for (auto character : text)
```

Reply:
you are right, thank you!